utils/util_carto.py

```diff
@@ -17,8 +20,96 @@ def upload_to_carto(file, privacy):
     # upload dataset to carto
     dataset = dataset_manager.create(file)
     logger.info('Carto table created: {}'.format(os.path.basename(file).split('.')[0]))
-    # set dataset privacy to 'Public with link'
+    # set dataset privacy
+    dataset.privacy = privacy
+    dataset.save()
+
+def create_carto_schema(gdf):
+    '''
+    Function to create a dictionary of column names and data types
+    in order to the upload the data to Carto
+    INPUT   gdf: geodataframe storing the data (dataframe)
```

Comment:
Does this have to be a geodataframe or could it be a regular dataframe?
```suggestion
    INPUT   gdf: geodataframe storing the data (dataframe)
```

Reply:
```suggestion
    INPUT   gdf: dataframe storing the data (dataframe)
```